keras/src/backend/common/variables.py

```diff
@@ -30,18 +30,17 @@ class KerasVariable:
             dtype type (`"float32"` if never configured).
         trainable: Optional. Boolean indicating if variable is trainable.
             Defaults to `True`.
+        autocast: Optional. Boolean indicating whether the variable supports
+            autocasting. If `True`, the layer may first convert the variable
+            to the compute data type when accessed. Defaults to `True`.
+        aggregation: Optional. String specifying how a distributed variable will
+            be aggregated. This argument is specific to TensorFlow. For more
+            details, please refer to
+            https://www.tensorflow.org/api_docs/python/tf/VariableAggregation.
+            Defaults to `"mean"`.
         name: Optional. A unique name for the variable. Automatically generated
             if not set.
 
-    Attributes:
```

Comment:
This section is useful, no?

Reply:
I deleted them because vscode w/ pylance mixed them into the arguments. However, they are undoubtedly useful.
Reverted.